src/MangaStreamParser.ts

```diff
@@ -28,9 +27,9 @@ export class MangaStreamParser {
             titles.push(this.decodeHTMLEntity(title.trim()))
         }
 
-        const author = $(`span:contains(${source.manga_selector_author}), .fmed b:contains(${source.manga_selector_author})+span, .imptdt:contains(${source.manga_selector_author}) i`).contents().remove().last().text().trim() // Language dependant
-        const artist = $(`span:contains(${source.manga_selector_artist}), .fmed b:contains(${source.manga_selector_artist})+span, .imptdt:contains(${source.manga_selector_artist}) i`).contents().remove().last().text().trim() // Language dependant
-        const image = this.getImageSrc($('img', 'div[itemprop="image"]'))
+        const author = $(`span:contains(${source.manga_selector_author}), .fmed b:contains(${source.manga_selector_author})+span, .imptdt:contains(${source.manga_selector_author}) i, .tsinfo > div:nth-child(4) > i`).contents().remove().last().text().trim() // Language dependant
+        const artist = $(`span:contains(${source.manga_selector_artist}), .fmed b:contains(${source.manga_selector_artist})+span, .imptdt:contains(${source.manga_selector_artist}) i, .tsinfo > div:nth-child(5) > i`).contents().remove().last().text().trim() // Language dependant
+        const image = this.getImageSrc($('img', 'div[itemprop="image"]'), undefined)
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
> Some mangastream sites use src attributes like "https://google.com/images/abc.png" and others '/images/abc.png".
I thought it might be a nice feature to have an already implemented system might we add some of these mangastream sources in the future.